scripts/prepare-release.sh

```diff
@@ -64,6 +66,10 @@ update_csv() {
 update_base_csv() {
   yq e -i ".metadata.name=\"$OLM_TYPE.v$VERSION\"" config/manifests-$OPERATOR_TYPE/bases/$OLM_TYPE.clusterserviceversion.yaml
   yq e -i ".spec.version=\"$VERSION\"" config/manifests-$OPERATOR_TYPE/bases/$OLM_TYPE.clusterserviceversion.yaml
+  if [[ "${VERSION}" != "${PREVIOUS_VERSION}" ]]; then
+    # yq e -i ".spec.replaces=\"${OLM_TYPE}.${PREVIOUS_VERSION}\"" bundles/$OLM_TYPE/${VERSION}/manifests/$OLM_TYPE.clusterserviceversion.yaml
```

Comment:
Is this commented out line included for a reason?

Reply:
No reason, forgot about removing it when I shifted to the replaces field being set in the config base of the operator! 
Thank you